### DISCUSSION TITLE
Octane - Low performance gain and high + inconsistent latency

### DISCUSSION THREAD

**Discussion by tarkis** [laravel/framework#37779] [General]
If someone are facing low performance gain or high and inconsistent latency double check middleware list of your route group. During this test I haven't modify any of it and because it used ``routes/web.php`` it created N number of session files that throttled the system after multiple runs.

**Comment by robert-moore96** [comment]
I havent ran Octane myself yet so not actually sure but i am wondering why you disabled OPCache for the octane tests?

also possibly related, you are letting fpm run up to 50/100 children, but for octane you are setting 8/16, your test script looks to be running 100 concurrent users so you are likely completely saturating (basically a DoS) the octane workers
